contracts/optimistic-ethereum/libraries/resolver/Lib_AddressResolver.sol

```diff
@@ -13,7 +13,7 @@ abstract contract Lib_AddressResolver {
      * Contract Variables: Contract References *
      *******************************************/
 
-    Lib_AddressManager internal libAddressManager;
+    Lib_AddressManager public libAddressManager;
```

Comment:
This is actually very security critical. Some of our (annoying) proxies require that we call `initialize()` outside of the constructor. Making this public allows us to check that the address has been set correctly when `initialize()` is called.

Reply:
This needs to be in a separate PR so that this one only touches hardhat deploy related things

Reply:
Ok I'll make a quick PR for this.

Reply:
PR is open here: https://github.com/ethereum-optimism/contracts/pull/352

Reply:
Done.